taxonomies/Message.ttl

```diff
@@ -60,6 +60,16 @@ ids:rejectionReason a owl:ObjectProperty;
     rdfs:label "rejectionReason"@en;
     rdfs:comment "Specifies the reason of the rejection."@en.
 
+# CallBack URL
+# ------------
+
+ids:callbackURL a owl:DatatypeProperty;
+    rdfs:label "callback URL"@en;
+    rdfs:comment "A callback URL must be provided in a request message and a notification message. "@en;
+    rdfs:domain [ owl:unionOf (ids:RequestMessage ids:NotificationMessage )];
+    rdfs:range xsd:anyURI.
```

Comment:
As this may not be a URL but any kind of address information for an abitrary system, maybe we could use a string?

Reply:
Since I am not an expert in this domain, @DErCouch @alexandrudanciu   can you please confirm this?

Reply:
@juliapampus could you please give an example where `xsd:anyURI` is not suitable? It covers all web-based communication protocols, including http, ftp, etc.

Reply:
@JohannesLipp , I don't receive a response from @juliapampus yet, therefore, I am not sure how to proceed.

Reply:
@changqin26 I could only imagine using the more general Internationalized Resource Identifier (IRI) insead of URLs. However, I don't acknowledge such a requirement. Regarding the name of the attribute I support using callbackAddress instead of callbackURL as we would use the type anyURI and not anyURL.